### DISCUSSION TITLE
Windows support?

### DISCUSSION THREAD

**Discussion by keichan34** [unvt/charites#182] [General]
Recently I've been using Windows as my main OS, and the WSL support is great. There are problems with supporting windows natively in charites (mainly, sporadic test failures) -- should we drop Windows completely and direct people to use WSL in a Windows environment, or should we work on getting the tests stable on Windows?

https://github.com/unvt/charites/issues/77

**Comment by smellman** [comment]
@keichan34 
Charites supports windows but CI is unstable now.
I don't remember the reason because it is not on history.
But you can try CI again in your branch.
